tests/unit/multieditor/test_actions.py

```diff
@@ -0,0 +1,527 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, print_function, division
+import json
+import os
+
+from inspirehep.modules.multieditor import actions
+from inspirehep.modules.multieditor.actions import Addition, Deletion, Update
+
+schema_1 = {
+  "properties": {
+      "abstracts": {
+        "description": ":MARC: ``520``",
+        "items": {
+          "additionalProperties": 'false',
+          "description": "This is used to add, besides the `value`, the `source`\
+                          where this value\ncame from.",
+          "properties": {
+            "source": {
+              "$schema": "http://json-schema.org/schema#",
+              "type": "string"
+            },
+            "value": {
+              "type": "string"
+            }
+          },
+          "required": [
+            "value"
+          ],
+          "type": "object"
+        },
+        "title": "List of abstracts",
+        "type": "array",
+        "uniqueItems": 'true'
+      }},
+  "type": "object",
+}
+
+curr_path = os.path.dirname(__file__)
+with open(os.path.join(curr_path, 'fixtures/schema.json')) \
+        as data_file:
+    schema = json.load(data_file)
```

Comment:
If this is needed, it should be converted into a fixture that then is added as a parameter to the test that needs it.